ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -327,6 +323,17 @@ function PoliciesTable({
                                                 Disable notification
                                             </DropdownItem>,
                                             <DropdownSeparator key="Separator" />,
+                                            <DropdownItem
+                                                key="Export policy"
+                                                component="button"
+                                                isDisabled={selectedPolicies.length < 1}
```

Comment:
Optional comment that `selectedPolicies.length === 0` seems clearer.

Array `length` property never has negative value, because it is controlled by JavaScript. For example, it throws an error if code assigns to negative value.